packages/design-system/src/__theme__/default/index.css

```diff
@@ -119,6 +121,8 @@
     * ===========================================*
   */
   --ads-v2-border-radius: 4px;
+  --ads-v2-pill-border-radius: 10px;
```

Comment:
Do you see any other use case for this variable? other than switch?

Reply:
not particularly but wasn't sure about hardcoding it - shall I?

Reply:
Yeah seems like this is only foe switch component. So keep it inside that component only